base/src/main/java/org/aya/core/repr/AyaShape.java

```diff
@@ -36,15 +40,17 @@ record AyaIntLitShape() implements AyaShape {
   }
 
   record AyaListShape() implements AyaShape {
+    public static final @NotNull CodeShape.MomentId NIL = new CodeShape.MomentId();
+    public static final @NotNull CodeShape.MomentId CONS = new CodeShape.MomentId();
```

Comment:
Why not just use the all-in-one enum directly?

Reply:
Good point